src/database/migrations/0012-create-platform_provider_responses.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE platform_provider_responses (
```

Comment:
Wanted to make sure you saw this thread: https://github.com/PhilanthropyDataCommons/service/pull/373#discussion_r1203042236

I agree / do prefer surrogate keys in general (as you know since we talked about), that said I think Jason's summary captures the reason why this is maybe different enough to be, well, different?

Jason also linked to an article (which I haven't dug into yet but plan to) that I think articulates all the reasons why my surrogate key bias is wrong 😂

(I don't remember what your preferences on this were in the original discussion)

Reply:
Whoops I had missed it. And I am OK with revisiting assuming others agree.

Reply:
I'm fine with revisiting later -- for this I'm inclined to just leave as currently is, which reinforces the "this is a cache" decision.